kubetest2-gke/deployer/options/cluster.go

```diff
@@ -45,6 +45,11 @@ type ClusterOptions struct {
 	WindowsMachineType string `flag:"~windows-machine-type" desc:"For use with gcloud commands to specify the machine type for Windows node in the cluster."`
 	WindowsImageType   string `flag:"~windows-image-type" desc:"The Windows image type to use for the cluster."`
 
+	SecondNodePoolEnabled     bool   `flag:"~enable-second-nodepool" desc:"Whether an extra node pool is created in the cluster."`
+	SecondNodePoolNumNodes    int    `flag:"~second-nodepool-num-nodes" desc:"For use with gcloud commands to specify the number of nodes for extra nodepool node pools in the cluster."`
+	SecondNodePoolMachineType string `flag:"~second-nodepool-machine-type" desc:"For use with gcloud commands to specify the machine type for extra nodepool in the cluster."`
+	SecondNodePoolImageType   string `flag:"~second-nodepool-image-type" desc:"The image type to use for the extra nodepool"`
+
```

Comment:
I'd want to validate multiple types of node pools... can this be structured in a way that would allow repeating the flag once for each node pool type we want to validate?

--extra-node-pool=num-nodes=1,machine-type=mytype,image-type=myimage
--extra-node-pool=num-nodes=1,machine-type=mytype2,image-type=myimage2